db/data_migration/20150317165803_republish_case_studies_to_render_on_government_frontend.rb

```diff
@@ -0,0 +1 @@
+DataHygiene::PublishingApiRepublisher.new(CaseStudy.where(:state => Edition::PUBLICLY_VISIBLE_STATES)).perform
```

Comment:
Rebased to be much much simpler with @evilstreak's help.

I'm also unsure about to handle `coming_soon` items. Are there likely to be any? I suppose a coming soon case study would only be broken if someone schedules one before this code change is deployed and we remove the support for coming soon items from Whitehall before they are actually published. This seems unlikely.